server/README.md

```diff
@@ -9,16 +9,23 @@
 ## Development
 
 ```
-# copy file and set proper data inside
-cp .env.example .env
+# Environmental variables
+
```

Comment:
Go into `server` folder and then:

Reply:
Ahh.. never mind - you are in server folder :D

Reply:
done